test/config-luatest/upgrade_test.lua

```diff
@@ -71,14 +71,13 @@ end)
 g.test_upgrade = function()
     g.server:exec(function()
         t.assert_equals(box.space._schema:get("version"), {'version', 2, 11, 0})
-        local messages = {}
-        for _, alert in pairs(require("config"):info().alerts) do
-            table.insert(messages, alert.message)
-        end
-        local exp = 'box.schema.user.grant("guest", "read", "space", ' ..
-            '"_space") has failed because either the object has not been ' ..
-            'created yet, a database schema upgrade has not been performed, ' ..
-            'or the privilege write has failed (separate alert reported)'
-        t.assert_items_include(messages, {exp})
+
+        -- We will change 'guest' rights, so need to change user
+        box.session.su('admin')
```

Comment:
`box.session.su` is not needed anymore (after changing the implementation to one that performs DDL from the background fiber).

Reply:
Oh. I see. Thank you. Applied.